transcom/mymove
# B-17690-Fix-Progear-Allowance

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

would like the Allowances screen to reflect the pro-gear amount entered by the service member,

So that I do not need to manually edit the allowances in order to see the information.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access milmovelocal
2. Create a PPM with Pro Gear
3. Access officelocal
4. Login as a TOO/SC
5. Open the PPM created with Pro Gear, under allowances should list 2000 for pro gear and 500 for spouse pro gear

### Backend
